validator/schemas/summary-metrics-schema-v1.json

```diff
@@ -138,9 +138,7 @@
         "eventId": "entity.guid"
       }],
       "required": [
-        "select",
-        "from",
```

Comment:
Nice, it will match what we have in GMs now: https://github.com/newrelic/entity-definitions/blob/828b9b663ac3759bb17bbd93f9496e4418ed1c95/docs/golden_metrics.md#query-options